extras/userdoc/md/documentation/installation.md

```diff
@@ -151,6 +151,11 @@ The easiest way to install NEST on a Mac is to install it via the Homebrew packa
 Options have to be appended, e.g. to install NEST with PyNEST run
 
     brew install nest --with-python
+    
+This will install the most recent release version of NEST. To build
+NEST from the most recent sources on Github, use
+
+    brew install nest --HEAD
```

Comment:
See https://github.com/brewsci/homebrew-science/issues/73 - does not work. Also with nest on homebrew-core the option `--HEAD` will not be provided, see https://github.com/Homebrew/homebrew-core/pull/21561#pullrequestreview-82487750 (open `Show outdated`).